### DISCUSSION TITLE
Vue Router page transitions not triggering...

### DISCUSSION THREAD

**Discussion by michealroberts** [nuxt/nuxt#17989] [Questions]
Hi, just wondering ... with the latest version of Nuxt, is anyone else experience issues with route changes not being triggered? The history seems to change, but the page stays at the same scroll position and no transitions are fired?

Is this something to be fixed?

**Comment by lucpotage** [comment]
For the scroll position, it seems to be the default behavior. You can change it via a plugin. Look at this comment: https://github.com/nuxt/framework/issues/1707#issuecomment-961983939
For page transiions, simply add transition style starting with `page-` and it should work. See comment: https://github.com/nuxt/framework/discussions/851#discussioncomment-1636922

**Comment by BuildTestRich** [comment]
I have this exact problem, any links that resolve to the same dynamic route don't work since updating to latest version.

For instance, two pages `/foo/bar` and `foo/baz` that both use the dynamic route `/foo/[slug].vue` can't link to each other using `<NuxtLink>` as they don't trigger a route change.

But links to any static route or different dynamic route work fine. The url is changing but no `page:start` hook is being triggered.
